### DISCUSSION TITLE
Define a validation process for PK-Sim multiple specie gene expression databases

### DISCUSSION THREAD

**Discussion by HenrikCordes** [Open-Systems-Pharmacology/Forum#1771] [Feature Requests & Ideas]
Dear Community, 

the latest [animal & human gene expression databases](https://github.com/Open-Systems-Pharmacology/Gene-Expression-Databases/releases ) are currently not part of the regular releases of PK-Sim.

As discussed previously #1623 a validation process should be in place that ensures the containing information (expression values & meta information) is correctly represented as in the source.

In addition, there might be PK-Sim updates/new features needed better enable the usage of the gene expression values:
- Animal expression databases can be queried for human homologs (e.g. CYP3A4, OATP1B1) the databases also contain the % sequence homology which could be used as indicator for an appropriate homolog
- Genes/protein annotation can also contain the subcellular location (intracellular, membrane, extracellular) which currently needs to be entered manually after the expression values are transferred from the databases 
- Amounts/concentrations of the gene/proteins are currently not retrieved from the expression databases 
- The expression databases contain information from multiple sources, thus the expression per organ is rather a distribution than a single value. Currently the expression distribution from the databases cannot be used for population/uncertainty simulations. 

We would appreciate your community input to this topic.

Best
Henrik

**Comment by StephanSchaller** [comment]
We could meet with the OMICS Focus Group?
